src/clj/rems/api/public.clj

```diff
@@ -56,7 +56,7 @@
 
     (GET "/full" []
       :summary "Get (almost) full configuration"
-      :roles #{:owner}
+      :roles #{:owner :organization-owner}
```

Comment:
this probably shouldn't be available to `:organization-owner` since it has info relating to the whole installation (e.g. oidc client secret)

Reply:
Done.